doc/sphinx/source/glossary.rst

```diff
@@ -25,8 +25,9 @@ Glossary
 
    framework
 
-      A machine learning environment, such as TensorFlow, MXNet, or
-      neon.
+      The software component built for the ingestion of data in a 
```

Comment:
The scope of a framework is broader than `ingestion of data`. Not sure how to define it.

Reply:
Yeah, since we refer to "frameworks" so much in our documentation, it's worthwhile to try to get this more accurate... especially if we have a bunch of new users coming in from the `pip install` route.  Updated the def slightly; lmk if it needs further refining.

Reply:
I think there's still too much emphasis on data ingestion here. Maybe something like: frameworks provide expressive user-facing APIs for constructing, training, validating, and deploying DL/ML models.